sdk/core/core-http/CHANGELOG.md

```diff
@@ -1,8 +1,18 @@
 # Release History
 
 ## 1.0.3 (Unreleased)
```

Comment:
Should we add the date here now that we know that we are releasing on Thursday?

```suggestion
## 1.0.3 (2020-01-02)
```

Reply:
Sure